core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -286,11 +286,11 @@ <h3>Roles</h3>
                   </span>
                 </div>
 
-                <input type="submit" class="btn btn-default protractor-test-save-role" value="Save">
-                <button type="button" class="btn btn-default" ng-click="closeEditRolesForm()"
-                        ng-show="isRolesFormOpen">
+                <input type="submit" class="btn btn-default protractor-test-save-role" value="Save" ng-disabled="!isTitlePresent()">
+                <button type="button" class="btn btn-default" ng-click="closeEditRolesForm()" ng-show="isRolesFormOpen">
                   Cancel
                 </button>
+                <p ng-show="!isTitlePresent()" class="text-danger  protractor-test-title-warning">Please provide a title before inviting.</p>
```

Comment:
There are 2 spaces between the classes, can you make it a single space?

Reply:
Done.